src/main/java/io/quarkus/search/app/fetching/QuarkusIO.java

```diff
@@ -30,17 +31,22 @@
 
 public class QuarkusIO implements AutoCloseable {
 
+    private static final URI QUARKUS_IO_URL_BASE = URI.create("https://quarkus.io");
```

Comment:
Maybe we can add it as an application property and then it'll still work for the local development as well?

Reply:
Right, will do